Mathlib/Combinatorics/SimpleGraph/Connectivity.lean

```diff
@@ -2635,9 +2678,19 @@ instance : Decidable G.Connected := by
   rw [connected_iff, ← Finset.univ_nonempty_iff]
   infer_instance
 
+open Finset in
+instance Path.instFintype {u v : V} : Fintype (G.Path u v) where
+  elems := (univ (α := { p : G.Walk u v // p.IsPath ∧ p.length < Fintype.card V })).map
```

Comment:
...this
```suggestion
  elems := (univ (α := { p : G.Walk u v | p.IsPath ∧ p.length < Fintype.card V })).map
```

Reply:
Yup, that worked! :) 

Any idea why one works better than the other?

Reply:
`{p : G.Walk u v | p.IsPath ∧ p.length < Fintype.card V}` is a set, `{p : G.Walk u v // p.IsPath ∧ p.length < Fintype.card V}` is a subtype. You were using the latter as a set, which doesn't really work (well, it does, but only if you spend some effort convincing Lean so)